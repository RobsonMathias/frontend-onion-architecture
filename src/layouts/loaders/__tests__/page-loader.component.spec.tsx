/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import React from 'react';

import { PageLoader } from '../page-loader.component';

describe('PageLoader', () => {
  it('should render successfully', () => {
    const { container } = render(<PageLoader />);
    expect(container).toMatchSnapshot();
  });

  it('when brand is true should render successfully', () => {
    const { container } = render(<PageLoader brand={true} />);
    expect(container).toMatchSnapshot();
  });
});
