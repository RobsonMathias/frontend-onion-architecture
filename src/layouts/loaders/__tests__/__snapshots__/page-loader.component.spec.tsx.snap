// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoader should render successfully 1`] = `
<div>
  <div
    class="MuiBox-root css-11rfm39"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorSecondary css-1szo49z-MuiCircularProgress-root"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </span>
  </div>
</div>
`;

exports[`PageLoader when brand is true should render successfully 1`] = `
<div>
  <div
    class="MuiBox-root css-12iza5x"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorSecondary css-1szo49z-MuiCircularProgress-root"
      role="progressbar"
      style="width: 40px; height: 40px;"
    >
      <svg
        class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </span>
  </div>
</div>
`;
